app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1387,12 +1459,28 @@ class BrowserTabFragment :
             is Command.ScreenLock -> screenLock(it.data)
             is Command.ScreenUnlock -> screenUnlock()
             is Command.ShowFaviconsPrompt -> showFaviconsPrompt()
+            is Command.ShowWebPageTitle -> showWebPageTitleInCustomTab(it.title, it.url)
             else -> {
                 // NO OP
             }
         }
     }
 
+    private fun showWebPageTitleInCustomTab(title: String, url: String?) {
+        if (isActiveCustomTab()) {
```

Comment:
this logic can be done in the ViewModel (except, `fragmentIsVisible()`) then it could also be tested that the Command is sent when the custom tab is active